eng/common/TestResources/setup-az-modules.yml

```
@@ -26,7 +26,7 @@ steps:
       targetType: inline
       pwsh: true
       script: |
-        . "eng/common/TestResources/Update-PSModulePathForHostedAgent.ps1"
+        eng/common/TestResources/Import-AzModules.ps1
```

I believe PS will require you to specify the relative path. 
```suggestion
        ./eng/common/TestResources/Import-AzModules.ps1
```

---

Tests passed with the current code so it seems fine?

---

But I can prefix with `./` if that is our convention

---

If it works I don't mind but I thought for sure PS didn't correctly resolve paths like this.

---

It definitely works, and in fact I copied it from another place in these scripts:

https://github.com/Azure/azure-sdk-for-net/pull/16117/files#diff-e4d8bce7ce67b80dcfd8a2c5b70b730b0b014a1e68bd3b881d16a2717808f450L28